src/main/java/com/conveyal/r5/transit/TransitLayer.java

```diff
@@ -682,18 +693,23 @@ public static TransitModes getTransitModes(int routeType) {
     /**
      * @return a semi-shallow copy of this transit layer for use when applying scenarios.
      */
-    public TransitLayer scenarioCopy(TransportNetwork newScenarioNetwork) {
+    public TransitLayer scenarioCopy(TransportNetwork newScenarioNetwork, boolean willBeModified) {
```

Comment:
When would willBeModified be false? When a scenario only affects the street layer?